### PR TITLE
Document list tests

### PR DESCRIPTION
**Pull request by AvihayMReblaze**

### CODE REVIEW

File: src/doc-editors/SecurityPoliciesEditor.vue [hunk 1]
```diff
@@ -537,7 +537,7 @@ export default defineComponent({
           this.localDoc.map[this.mapEntryIndex].match = this.initialMapEntryMatch
         }
         this.mapEntryIndex = (this.mapEntryIndex === index ? -1 : index)
-        Utils.clearInputValidationClasses(this.$refs.mapEntryMatch[0])
+        Utils.clearInputValidationClasses(this.$refs. mapEntryMatch[0])
```
Comment (Aviv-Galmidi) [inline src/doc-editors/SecurityPoliciesEditor.vue:540]: Accidental whitespace?

File: src/views/__tests__/MainComponent.spec.ts [hunk 2]
```diff
@@ -1,31 +0,0 @@
-// @ts-nocheck
```
Comment (Aviv-Galmidi) [inline src/views/__tests__/MainComponent.spec.ts:1]: We need this file, please return it to the source code

File: vue.config.js [hunk 3]
```diff
@@ -3,7 +3,6 @@ const StyleLintPlugin = require('stylelint-webpack-plugin')
 
 module.exports = {
   devServer: {
-    // proxy: 'http://localhost:5000'
```
Comment (Aviv-Galmidi) [inline vue.config.js:6]: Please return this line, it is being used to let the developers know where the proxy for the confserver is located

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]
Please remove the vscode and similar files that are not needed from the git source code:
`.DS_store` file
`.vscode` directory
`mocks` directory
`ts-shim.d.ts` (from `src` and `components`)
`webpack.config.js`

File: vue.config.js (hunk 3 above)
Comment (AvihayMReblaze) [reply vue.config.js:6]: done

File: src/doc-editors/SecurityPoliciesEditor.vue (hunk 1 above)
Comment (AvihayMReblaze) [reply src/doc-editors/SecurityPoliciesEditor.vue:540]: Accident, done.

File: src/views/__tests__/MainComponent.spec.ts (hunk 2 above)
Comment (AvihayMReblaze) [reply src/views/__tests__/MainComponent.spec.ts:1]: done

**Review by Aviv-Galmidi — APPROVED** [review]

**Pull request merged**